.github/CONTRIBUTING.md

```diff
@@ -12,15 +12,16 @@ style, we make use of the [pre-commit](https://pre-commit.com/) framework to run
 several *hooks* when committing code. Please follow the instructions below if
 you intend to contribute to the lmfit repository:
 
-- clone the GitHub repository:
-  ``git clone https://github.com/lmfit/lmfit-py.git``
+- fork the GitHub repository
+- clone your forked GitHub repository:
+  ``git clone https://github.com/<your-name>/lmfit-py.git``
 - install all (optional) dependencies either using ``pip`` or ``conda``:
-  ``pip -r install requirements-dev.txt`` or
+  ``pip install -U -r requirements-dev.txt`` or
   ``conda install <packages in requirements-dev.txt>``
 - initialize ``pre-commit`` running ``pre-commit install`` in the lmfit directory
 - create a new branch: ``git checkout -b <awesome_new_feature>``
 - start coding
-- make sure the test-suite passes locally: run ``pytest`` in the lmfit directory
+- make sure the test-suite passes locally: run ``python -m pytest`` in the lmfit directory
```

Comment:
let's add this line after the test-suite one (since we ask this in the PR checklist anyway):
- make sure that the documentation builds locally: run ``make`` in the ``doc`` directory